### ISSUE TITLE
missing closing cffunction tag not detected

### ISSUE THREAD

**Issue by zspitzer** [cflint/CFLint#468]
```<cfcomponent>
<cffunction name="testFunction" displayname="" access="public" output="false" returntype="void" hint="">
</cfcomponent>
```
